plugins/modules/aci_access_port_block_to_access_port.py

```diff
@@ -142,7 +157,7 @@
     state: present
   delegate_to: localhost
 
-- name: Remove an access port block from an interface selector
+- name: Remove an Fabric access policy interface port block from an interface selector
```

Comment:
Can you move the query examples before remove?

Reply:
Yes, will do it.